src/App.tsx

```diff
@@ -1,7 +1,32 @@
-import React from 'react';
+import { Component } from 'react';
+import './index.scss';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
```

Comment:
1. State has to be created in constructor
2. String `'Nothing was pressed yet'` is not the key type, so I suggest to set `pressedKey: null` by default and then make condition in your JSX
```suggestion
  constructor(props) {
    super(props);
    this.state = { pressedKey: null },
  }
```